tests/_utils.py

```diff
@@ -101,3 +103,38 @@ def _base_problem_type(self) -> Type[B]:
     @property
     def _valid_policies(self) -> Tuple[str, ...]:
         return ()
+
+
+def create_lr_initializer(
+    initializer,
+    rank,
+    **kwargs,
+) -> lr_init_lib.LRInitializer:  # noqa: D102
+    if isinstance(initializer, lr_init_lib.LRInitializer):
+        return initializer
+    if initializer == "random":
+        return lr_init_lib.RandomInitializer(rank=rank, **kwargs)
+    if initializer == "rank2":
+        return lr_init_lib.Rank2Initializer(rank=rank, **kwargs)
+    if initializer == "k-means":
+        return lr_init_lib.KMeansInitializer(rank=rank, **kwargs)
+    if initializer == "generalized-k-means":
+        return lr_init_lib.GeneralizedKMeansInitializer(rank=rank, **kwargs)
+    raise NotImplementedError(f"Initializer `{initializer}` is not yet implemented.")
+
+
+def create_fr_initializer(
+    initializer,
+    **kwargs,
+) -> init_lib.SinkhornInitializer:  # noqa: D102
+    if isinstance(initializer, init_lib.SinkhornInitializer):
+        return initializer
+    if initializer == "default":
+        return init_lib.DefaultInitializer(**kwargs)
+    if initializer == "gaussian":
+        return init_lib.GaussianInitializer(**kwargs)
+    if initializer == "sorting":
+        return init_lib.SortingInitializer(**kwargs)
+    if initializer == "subsample":
+        return init_lib.SubsampleInitializer(**kwargs)
+    raise NotImplementedError(f"Initializer `{initializer}` is not yet implemented.")
```

Comment:
Can we keep strings for initializers , or do `Literal[...], init_lib.SinkhornInitializer` and port the code to src?